exercises/acronym/canonical-data.json

```diff
@@ -60,6 +60,14 @@
             "phrase": "Something - I made up from thin air"
           },
           "expected": "SIMUFTA"
+        },
+        {
+          "description": "apostrophes",
+          "property": "abbreviate",
+          "input": {
+            "phrase": "Haley's Comet"
```

Comment:
If I may -

If this is to be representative of a real-world phrase, then I would suggest using `Halley's Comet` here with the extra L. 

However, if all resemblances to real-world phenomena are purely coincidental, then as you were.